packages/compose/src/hooks/use-drop-zone/index.js

```diff
@@ -73,18 +73,24 @@ export default function useDropZone( {
 			/**
 			 * Checks if an element is in the drop zone.
 			 *
-			 * @param {HTMLElement|null} elementToCheck
+			 * @param {EventTarget|null} targetToCheck
 			 *
 			 * @return {boolean} True if in drop zone, false if not.
 			 */
-			function isElementInZone( elementToCheck ) {
+			function isElementInZone( targetToCheck ) {
+				const { defaultView } = ownerDocument;
 				if (
-					! elementToCheck ||
-					! element.contains( elementToCheck )
+					! targetToCheck ||
+					! defaultView ||
+					! ( targetToCheck instanceof defaultView.HTMLElement ) ||
+					! element.contains( targetToCheck )
 				) {
 					return false;
 				}
 
+				/** @type {HTMLElement|null} */
```

Comment:
Good question! The answer is yes, there needs to be margin for `null` for the subsequent do/while loop. Otherwise, TS complains at `elementToCheck = elementToCheck.parentElement`.

Reply:
Ah, I see, I missed that it's a `let` and not a `const`. 👍